_install-and-configure/install-opensearch/index.md

```diff
@@ -17,7 +17,17 @@ This section details how to install OpenSearch on your host, including which ope
 
 ## Operating system compatibility
 
-We recommend installing OpenSearch on Red Hat Enterprise Linux (RHEL) or Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, or Ubuntu Long-Term Support (LTS). OpenSearch should work on most Linux distributions, but we only test a handful. We recommend RHEL 7 or 8, CentOS 7 or 8, Amazon Linux 2, or Ubuntu 16.04, 18.04, or 20.04 for any version of OpenSearch. OpenSearch also supports Windows Server 2019.
+OpenSearch is compatible with Red Hat Enterprise Linux (RHEL) and Debian-based Linux distributions that use [systemd](https://en.wikipedia.org/wiki/Systemd), such as CentOS, Amazon Linux 2, or Ubuntu Long-Term Support (LTS). While OpenSearch should work on most Linux distributions, we only test a subset. 
+
+The following table lists the operating system versions that we support. 
```

Comment:
`that we support` to `that we currently support`?

Reply:
Curious - do you think we should backport this change all the way back to 1.3?

Reply:
There are a few issues here:
1. This does not only apply to RPM related OS, but apply to all OS that does not support sha1 key by default, so I think the download page would be a better idea.
2. The download page yum/apt download link is not correctly link to rpm/deb page.
3. DEB page is missing in the 1.3 section of documentation website installation pages.

Reply:
And yes I think we should also backport to 1.3, due to eventually 1.3 and 2.x might have different or same compatibilities that we dont know, and such a chart would help for sure.

Reply:
Thanks, Peter. Updates to the following:
- I added a link from the RPM page to the operating system compatibility table
- I added redirects to the RPM and Debian pages so the "install" link on the Downloads page takes the user directly to those pages

Fanit is going to fix the issue with the parent link on the 1.3 version so we'll be able to backport easily to 1.3. Once she fixes that, I'll backport all installation files, including Debian. The 'copy' buttons should be appearing, as well. 

I'll create an issue to fix the Downloads page:
- Add a link to the "Operating System compatibility" chart on the install page